src/kuzya.cpp

```diff
@@ -193,6 +193,19 @@ Kuzya::Kuzya(QWidget *parent)
     settings->readODWSettings();
     settings->openLastProject();
     settings->readMainWindowState();
+    tFile = new QTemporaryFile;
+    if(settings->getIsFileReopenEnabled())
```

Comment:
it better to use either "get" or "is" prefix for getters (I mean name it: getFileReopenState() or isFileReopenEnabled()). I suggest to use second one (isFileReopenEnabled()) - it is more in Qt fashion.